frontend/webadmin/modules/uicommonweb/src/main/java/org/ovirt/engine/ui/uicommonweb/models/clusters/ClusterModel.java

```diff
@@ -1522,6 +1522,7 @@ public void refreshMigrationPolicies() {
             selectedPolicyId = getMigrationPolicies().getSelectedItem().getId();
         }
 
+        // TODO: Check/disable the new policy when parallel migrations are not enabled in Engine config
```

Comment:
You probably cannot disable an item in the dropdown, but if the policy is not supported, you can remove it from the list before setting it 
```
policies.remove(policy)
getMigrationPolicies().setItems(policies);
```

Reply:
Good idea, thanks for the advice.

Reply:
Done.